pkg/subctl/cmd/join.go

```diff
@@ -71,6 +73,7 @@ func addJoinFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&ipsecDebug, "ipsec-debug", false, "Enable IPsec debugging (verbose logging)")
 	cmd.Flags().BoolVar(&submarinerDebug, "subm-debug", false, "Enable Submariner debugging (verbose logging)")
 	cmd.Flags().BoolVar(&noLabel, "no-label", false, "skip gateway labeling")
+	cmd.Flags().StringVar(&kubefedHostClusterContext, "kubefedHostClusterContext", "", "overrides the use of host-cluster-context name in resource names created in the target cluster.")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&kubefedHostClusterContext, "kubefed-host-cluster-context", "", "overrides the use of host-cluster-context name in resource names created in the target cluster.")
```

To follow CLI arg naming conventions.

Reply:
Oh just forgot to push the updated one. Done.